source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
+
+class WordChartTitle(WordChartElement):
+	def __init__(self,windowHandle, wordApplicationObject, wordChartObject, keyIndex, elementIndex):
+		self.chartTitle=wordChartObject.ChartTitle.Text
+		super(WordChartTitle,self).__init__(windowHandle=windowHandle, wordApplicationObject=wordApplicationObject, wordChartObject=wordChartObject, keyIndex=keyIndex, elementIndex=elementIndex)
+
+	def _get_name(self):
+		# Translators: Message reporting the chart title
+		text= _("Chart title: {chartTitle}").format(chartTitle=self.chartTitle)
+		return text
+
+class WordChartAxis(WordChartElement):
+	def __init__(self,windowHandle, wordApplicationObject, wordChartObject, axisType, axisGroup, keyIndex, elementIndex):
+		self.wordChartObject=wordChartObject
+		self.axisType=axisType
+		self.axisGroup=axisGroup
+		super(WordChartAxis,self).__init__(windowHandle=windowHandle, wordApplicationObject=wordApplicationObject, wordChartObject=wordChartObject, keyIndex=keyIndex, elementIndex=elementIndex)
+
+	def _get_name(self):
+		return self._axisMap[self.axisType][self.axisGroup]
+
+class WordChartAxisTitle(WordChartElement):
+	def __init__(self,windowHandle, wordApplicationObject, wordChartObject, axisType, axisGroup, keyIndex, elementIndex):
+		self.wordChartObject=wordChartObject
+		self.axisType=axisType
+		self.axisGroup=axisGroup
+		super(WordChartAxisTitle,self).__init__(windowHandle=windowHandle, wordApplicationObject=wordApplicationObject, wordChartObject=wordChartObject, keyIndex=keyIndex, elementIndex=elementIndex)
+
+	def _get_name(self):
+		# Translators: Message reporting axis name and axis title
+		text= _("{axisName} title: {axisTitle}").format(axisName=self._axisMap[self.axisType][self.axisGroup], axisTitle=self.wordChartObject.Axes(self.axisType, self.axisGroup).AxisTitle.Text)
+		return text
+
+class WordChartSeries(WordChart):
+	def __init__(self,windowHandle, wordApplicationObject, wordChartObject, keyIndex, seriesIndex, pointIndex=0):
+		self.seriesIndex=seriesIndex
+		self.currentPointIndex=pointIndex
+		self.keyIndex=keyIndex
+		self.seriesCount=wordChartObject.SeriesCollection().Count
+		self.pointsCollection=wordChartObject.SeriesCollection(self.seriesIndex).Points()
+		self.pointsCount=self.pointsCollection.Count
+		super(WordChartSeries,self).__init__(windowHandle=windowHandle, wordApplicationObject=wordApplicationObject, wordChartObject=wordChartObject, keyIndex=keyIndex)
+
+	def _get_name(self):
+		currentSeries=self.wordChartObject.SeriesCollection(self.seriesIndex)
+		# Translators: Details about a series in a chart. For example, this might report "foo series 1 of 2"
+		seriesText=_("{seriesName} series {seriesIndex} of {seriesCount}").format( seriesName = self.wordChartObject.SeriesCollection(self.seriesIndex).Name , seriesIndex = self.seriesIndex , seriesCount = self.seriesCount )
+		return seriesText
+
+	def _get_role(self):
+		return controlTypes.ROLE_CHARTELEMENT

```

Comment:
Changed it to a class variable.
Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.